### PR TITLE
#9.2 Updated

### PR DESCRIPTION
**Pull request by Jahongir1507**

### CODE REVIEW

File: WeebApp/Areas/user/Controllers/PostController.cs [hunk 1]
```diff
@@ -35,7 +35,7 @@ public async Task<IActionResult> Index()
         {
             var curUserId = User.FindFirstValue(ClaimTypes.NameIdentifier);
 
-            var UserPost = _context.Posts.OrderByDescending(p => p.CreatedDate);
+            var UserPost = _context.Posts.OrderByDescending(p => p.CreatedDate).Take(8);
             var UserPost_2 = UserPost.Where(p => p.CreatorId == curUserId);
```
Comment (napster1991) [inline WeebApp/Areas/user/Controllers/PostController.cs]: 38-39 qatorlarni bir qatorda ham qilsa bo'ladi:
`var UserPost = _context.Posts.Where(p => p.CreatorId == curUserId).OrderByDescending(p => p.CreatedDate).Take(8);`

File: WeebApp/Areas/user/Views/Post/Create.cshtml [hunk 2]
```diff
@@ -23,8 +23,25 @@
                 <input asp-for="Text" class="form-control" />
                 <span asp-validation-for="Text" class="text-danger"></span>
             </div>
-          <button type="submit" class="btn btn-primary">Submit</button>
-        </form>
+            <br/>
+            <input type="submit" name="submitbtn" value="Create as draft" class="btn btn-primary" />
+            <input type="submit" name="submitbtn" value="Submit to check" class="btn btn-primary" />
+
+            @*    <span asp-validation-for="CreatedDate" class="text-danger"></span>
```
Comment (napster1991) [inline WeebApp/Areas/user/Views/Post/Create.cshtml]: commented out codeni remove qivorin

File: WeebApp/Areas/user/Views/Post/Index.cshtml [hunk 3]
```diff
@@ -44,10 +46,31 @@
                 @Html.DisplayFor(modelItem => item.CreatorId)
             </td>
             <td>
+                @Html.DisplayFor(modelItem => item.StatusId)
+            </td>
+          @*  <td>
```
Comment (napster1991) [inline WeebApp/Areas/user/Views/Post/Index.cshtml]: commented out codeni remove qivorin

File: WeebApp/Areas/user/Views/Post/Index.cshtml [hunk 4]
```diff
@@ -44,10 +46,31 @@
...
+                @Html.DisplayFor(modelItem => item.Status)
+            </td>*@
+           @* <td>
```
Comment (napster1991) [inline WeebApp/Areas/user/Views/Post/Index.cshtml]: commented out codeni remove qivorin

File: WeebApp/Controllers/HomeController.cs [hunk 5]
```diff
@@ -17,17 +19,17 @@ public class HomeController : Controller
...
         }
 
         [HttpGet]
-        public async Task<IActionResult> Index()
+
+        public IActionResult Index()
         {
-            var posts=_applicationDbContext.Posts.OrderByDescending(x=>x.CreatedDate).Take(8).ToList();
-            return View(posts); 
+            var posts = _applicationDbContext.Posts.OrderByDescending(p => p.CreatedDate).Where(p => p.StatusId == Enums.StatusEnum.Published).Take(8).ToList();
```
Comment (napster1991) [inline WeebApp/Controllers/HomeController.cs]: `var posts = _applicationDbContext.Posts.Where(p => p.StatusId == Enums.StatusEnum.Published).OrderByDescending(p => p.CreatedDate).Take(8).ToList();`

**Review by napster1991 — CHANGES_REQUESTED** [review]

File: WeebApp/Areas/user/Controllers/PostController.cs (hunk 1 above)
Comment (Jahongir1507) [reply WeebApp/Areas/user/Controllers/PostController.cs]: tug'irladim

File: WeebApp/Areas/user/Views/Post/Index.cshtml (hunk 3 above)
Comment (Jahongir1507) [reply WeebApp/Areas/user/Views/Post/Index.cshtml]: done

**Review by napster1991 — APPROVED** [review]

**Pull request merged**